tabs-tabs-tabs/tabs.js

```diff
@@ -1,23 +1,40 @@
+function firstUnpinnedTab(tabs) {
+  for (var tab of tabs) {
+    if (!tab.pinned) {
+      return tab.index;
+    }
+  }
+}
+
 document.addEventListener("click", function(e) {
   function callOnActiveTab(callback) {
-    chrome.tabs.query({}, function(tabs) {
+    chrome.tabs.query({windowId: chrome.windows.WINDOW_ID_CURRENT}, function(tabs) {
```

Comment:
You'd be better off using `currentWindow: true`. Firefox only supports `WINDOW_ID_CURRENT` as of 48.